cache/sampleStore.js

```diff
@@ -38,6 +38,11 @@ const constants = {
   objectType: { aspect: 'aspect', sample: 'sample', subject: 'subject' },
   prefix: PFX,
   separator: SEP,
+  commands: {
+    hgetall: 'hgetall',
+    hget: 'hget',
+    smembers: 'smembers',
+  },
 };
```

Comment:
Unless you are creating your own wrapper for the redis commands, there is little value exporting them.